docs/scenarios_authoring/RRDataSourcesAndSinks.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 4
+---
+
+# Request-Response sources and sinks
+
+## Http request as input and output to Nussknacker scenarios
+
+### Prerequisites
+
+To fully understand how Nussknacker works in Request-Response mode, it's best to read the following first:
+* [Http overview](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview)
+
+### Concepts
+
+This section discusses HTTP based source and sink components.
+
+Nussknacker can process single HTTP request as a scenario input. 
+In this case source and input are integrally connected meaning that the source component defines a response for incoming request for exact HTTP session.
+
+## Request-Response schema
+
+Each scenario needs to define two schemas:
+- Input schema
+- Output schema
+
+Nussknacker uses those schema to detect how the request and response should look like.
+
+![RR schema](img/rrProperties.png "RR properties")
+
+## Request-Response source
+
+Nussknacker will read HTTP Post request payload.
```

Comment:
I would move the stuff about "HTTP Post" to configuration or integration; the intention is not to overload people who author scenarios with technical details.